engine/resources/exec.go

```diff
@@ -368,6 +377,18 @@ func (obj *ExecRes) CheckApply(apply bool) (bool, error) {
 	defer cancel()
 	cmd := exec.CommandContext(ctx, cmdName, cmdArgs...)
 	cmd.Dir = obj.Cwd // run program in pwd if ""
+
+	envKeys := []string{}
+	for i := range obj.Env {
+		envKeys = append(envKeys, i)
+	}
+	sort.Strings(envKeys)
+	cmdEnv := []string{}
+	for k := range envKeys {
```

Comment:
Use the `for _, x := range` form and code will look nicer.

Reply:
do you want me to do this for all of my for loops?

Reply:
It depends on the case. In this case it pulls out the value automatically so you don't need to have it look more complex. Know what I mean?

Reply:
gotcha

Reply:
Not sure you did this...